eventlet/wsgi.py

```diff
@@ -8,12 +8,16 @@
 
 from eventlet.green import BaseHTTPServer
 from eventlet.green import socket
-from eventlet.green import urllib
 from eventlet import greenio
 from eventlet import greenpool
 from eventlet import support
 from eventlet.support import six
 
+if six.PY2:
```

Comment:
six does this `if PY2` check for you: `from six.moves.urllib.parse import unquote`

As much as I avoid importing module variables/functions, can't think of better option here.